File: HackerRank/problem3.py
'''
1. Declear an array size and input their values, 
2. create a plusMinus method where just pass 1 peramiter (arr), 
      In the method find out positive, negetive and zero 
3. find out the total positive, negetive and zero number and print their average !!
'''


def plusMinus(arr):
    positive = 0
    negative = 0
    zero = 0
    n = len(arr)

    for x in range(n):
        if arr[x]>0:
            positive+=1
        elif arr[x]<0:
            negative+=1
        elif arr[x]==0:
            zero+=1

    print(float(positive/n))
    print(float(negative/n))
    print(float(zero/n))



n = int(input("Enter Number : "))
arr = []

if n>0 and n<=100:
    for x in range(n):
        inputValue = int(input())
        if inputValue>=-100 and inputValue<=100:
            arr.append(inputValue)

print(arr)
plusMinus(arr)

File: HackerRank/problem3.cpp
#include<iostream>
using namespace std;


void plusMinus(int arr[]){
   int positive = 0;
   int negative = 0;
   int zero = 0;
   int n = sizeof(arr);

   for(int i=0; i<n; i++){
    if(arr[i]>0){
        positive++;
    }
    else if(arr[i]<0){
        negative++;
    }
    else if(arr[i]==0){
        zero++;
    }
   }

   cout<<(double(positive/n));
   cout<<(double(negative/n));
   cout<<(double(zero/n));
}

int main(){

   int n;
   int inputData;
   cin>>n;
   int arr[n];
   if(n>0 && n<=100){
    for(int i=0; i<n; i++){
        cin>>inputData;
        if(inputData>=-100 && inputData<=100){
            arr[i] = inputData;
        }
    }
    plusMinus(arr);
   }


    return 0;
}